src/components/chart/RadarChartCard.js

```diff
@@ -17,12 +17,11 @@ const RadarChartCard = props => {
           polygons: {
             strokeColor: '#e9e9e9',
             fill: {
-              colors: ['#f8f8f8', '#fff']
             }
           }
         }
       },
-      colors: ['#FF4560'],
+      colors: ['#546E7A', '#66DA26', '#546E7A', '#E91E63', '#FF9800'],
```

Comment:
this does not feel right, we need to check for something assigning a dinamic number of colours.